src/BusinessLogic/CheckoutAPI/PaymentMethods/Requests/GetCachedPaymentMethodsRequest.php

```diff
@@ -40,6 +58,38 @@ public function setMerchantId(string $merchantId): void
         $this->merchantId = $merchantId;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getShippingCountry(): ?string
+    {
+        return $this->shippingCountry;
+    }
+
+    /**
+     * @param string|null $shippingCountry
+     */
+    public function setShippingCountry(?string $shippingCountry): void
+    {
+        $this->shippingCountry = $shippingCountry;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getCurrentCountry(): ?string
+    {
+        return $this->currentCountry;
+    }
```

Comment:
```suggestion
     * @return string
     */
    public function getCurrentCountry(): string
    {
        return (string) $this->currentCountry;
    }
```
Related to https://github.com/sequra/integration-core/pull/38/files#r2399231555